react/src/pages/SslCertificatesPage.js

```diff
@@ -0,0 +1,28 @@
+import { useSearchParams } from 'react-router-dom';
+import useHeader from '../hooks/Header';
+import useFetch from '../hooks/Fetch';
+import SslCertificate from '../components/SslCertificate';
+
+const SslCertificatesPage = (props) => {
+    const [ args ] = useSearchParams();
+    const hostname = args.get("hostname");
+    const certificates = useFetch(`/certificates?hostname=${hostname}`);
```

Comment:
Should you need to pass a hostname argument? Can't the backend determine this automatically?

Reply:
It (the backend) does do this automatically, if a hostname argument is not specified (i.e. by default). What this is is, since it was easy to do, and was useful for testing, you can optionally pass a hostname argument to the page (and actually there is no UI do to this), and if specified it will return info on the specified certificate(s).

I.e. This shows certificate info for Foursight and Portal:
https://y1do9unbp3.execute-api.us-east-1.amazonaws.com/api/react/cgap-devtest/certificates

And this shows certificate info for pypi.org and slashdot.org:
https://y1do9unbp3.execute-api.us-east-1.amazonaws.com/api/react/cgap-devtest/certificates?hostname=pypi.org,slashdot.org